### PR TITLE
Remove python3_only and six from UTScapy

### PR DESCRIPTION
**Pull request by polybassa**

### CODE REVIEW

File: scapy/tools/UTscapy.py [hunk 1]
```diff
@@ -79,7 +87,7 @@ def retry_test(func):
             success = True
             break
     if not success:
-        six.reraise(t, v, tb)
+        reraise(t, v, tb)
```
Comment (gpotter2) [inline scapy/tools/UTscapy.py:90]:
```suggestion
        raise v.with_traceback(tb)
```
and remove `reraise`

File: scapy/tools/UTscapy.py [hunk 2]
```diff
@@ -177,12 +185,12 @@ class External_Files:
 /i7kinChIXSAmRgA==\n""")
 
     def get_local_dict(cls):
-        return {x: y.name for (x, y) in six.iteritems(cls.__dict__)
+        return {x: y.name for (x, y) in iter(cls.__dict__.items())
```
Comment (gpotter2) [inline scapy/tools/UTscapy.py:188]: iter unrequired.

File: scapy/tools/UTscapy.py [hunk 3]
```diff
@@ -177,12 +185,12 @@ class External_Files:
...
                 if isinstance(y, File)}
     get_local_dict = classmethod(get_local_dict)
 
     def get_URL_dict(cls):
-        return {x: y.URL for (x, y) in six.iteritems(cls.__dict__)
+        return {x: y.URL for (x, y) in iter(cls.__dict__.items())
```
Comment (gpotter2) [inline scapy/tools/UTscapy.py:193]: same

File: scapy/tools/UTscapy.py [hunk 4]
```diff
@@ -1129,7 +1121,7 @@ def main():
     KW_KO.append("disabled")
 
     # Process extras
-    if six.PY2 and DARWIN:
+    if DARWIN:
```
Comment (gpotter2) [inline scapy/tools/UTscapy.py:1124]: absolutely not !

File: scapy/tools/UTscapy.py [hunk 5]
```diff
@@ -1176,7 +1168,7 @@ def main():
     UNIQUE = len(TESTFILES) == 1
 
     # Resolve tags and asterix
-    for prex in six.iterkeys(copy.copy(PREEXEC_DICT)):
+    for prex in iter(copy.copy(PREEXEC_DICT).keys()):
```
Comment (gpotter2) [inline scapy/tools/UTscapy.py:1171]: iter required?

File: test/run_tests [hunk 6]
```diff
@@ -21,7 +21,6 @@ then
   do
     case $arg
     in
-      -2) PYTHON=python2;;
```
Comment (gpotter2) [inline test/run_tests:24]: I would rather change the script entirely, since `-3` isn't useful anymore.

File: test/run_tests.bat [hunk 7]
```diff
@@ -5,10 +5,7 @@ set PYTHONPATH=%MYDIR%
 REM Note: shift will not work with %*
 REM ### Get args, Handle Python version ###
 set "_args=%*"
-IF "%1" == "-2" (
-  set PYTHON=python
-  set "_args=%_args:~3%"
-) ELSE IF "%1" == "-3" (
```
Comment (gpotter2) [inline test/run_tests.bat:11]: same

**Review by gpotter2 — CHANGES_REQUESTED** [review]
A bunch of things I dislike

**Comment by gpotter2** [comment]
Thanks a lot for the PR @polybassa ! :stuck_out_tongue:

File: test/run_tests.bat (hunk 7 above)
Comment (polybassa) [reply test/run_tests.bat:11]: I'm not into batch file programming. Sorry! I don't feel competent to fix this file properly.

Comment (KhazAkar) [reply test/run_tests.bat:11]: Will try to check this batch file in my spare time how it can be improved

**Comment by gpotter2** [comment]
Hi ! could you please rebase against master? Thanks !

**Comment by polybassa** [comment]
@gpotter2 Done

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**